javaslang/src-gen/main/java/javaslang/API.java

```diff
@@ -84,6 +92,1316 @@ private API() {
+     *
+     * @param <K>   The key type.
+     * @param <V>   The value type.
+     * @param key   A singleton map key.
+     * @param value A singleton map value.
+     * @return A new LinkedHashMap instance containing the given entry
+     */
+    public static <K, V> LinkedHashMap<K, V> LinkedMap(K key, V value) {
+        return LinkedHashMap.of(key, value);
+    }
+
+    /**
+     * Alias for {@link LinkedHashMap#ofAll(Map)}
+     *
+     * @param <K> The key type.
+     * @param <V> The value type.
+     * @param map A map entry.
+     * @return A new LinkedHashMap instance containing the given map
+     */
+    public static <K, V> LinkedHashMap<K, V> LinkedMap(Map<? extends K, ? extends V> map) {
+        return LinkedHashMap.ofAll(map);
+    }
+
+    /**
+     * Alias for {@link HashMap#of(Object, Object)}
+     *
+     * @param <K>   The key type.
+     * @param <V>   The value type.
+     * @param key   A singleton map key.
+     * @param value A singleton map value.
+     * @return A new HashMap instance containing the given entry
+     */
+    public static <K, V> HashMap<K, V> Map(K key, V value) {
+        return HashMap.of(key, value);
+    }
+
+    /**
+     * Alias for {@link HashMap#ofAll(Map)}
+     *
+     * @param <K> The key type.
+     * @param <V> The value type.
+     * @param map A map entry.
+     * @return A new HashMap instance containing the given map
+     */
+    public static <K, V> HashMap<K, V> Map(Map<? extends K, ? extends V> map) {
+        return HashMap.ofAll(map);
+    }
+
+    /**
+     * Alias for {@link TreeMap#of(Comparable, Object)}
+     *
+     * @param <K>   The key type.
+     * @param <V>   The value type.
+     * @param key   A singleton map key.
+     * @param value A singleton map value.
+     * @return A new TreeMap instance containing the given entry
+     */
+    public static <K extends Comparable<? super K>, V> TreeMap<K, V> SortedMap(K key, V value) {
+        return TreeMap.of(key, value);
+    }
+
+    /**
+     * Alias for {@link TreeMap#of(Comparator, Object, Object)}
+     *
+     * @param <K>           The key type.
+     * @param <V>           The value type.
+     * @param keyComparator The comparator used to sort the entries by their key
+     * @param key           A singleton map key.
+     * @param value         A singleton map value.
+     * @return A new TreeMap instance containing the given entry
+     */
+    public static <K, V> TreeMap<K, V> SortedMap(Comparator<? super K> keyComparator, K key, V value) {
+        return TreeMap.of(keyComparator, key, value);
+    }
+
+    /**
+     * Alias for {@link TreeMap#ofAll(Map)}
+     *
+     * @param <K> The key type.
+     * @param <V> The value type.
+     * @param map A map entry.
+     * @return A new TreeMap instance containing the given map
+     */
+    public static <K extends Comparable<? super K>, V> TreeMap<K, V> SortedMap(Map<? extends K, ? extends V> map) {
+        return TreeMap.ofAll(map);
+    }
+
```

Comment:
This looks really great! When reading this file I can't wait to get my hands on the new API :-))

Please add also all empty collection methods:

* List(), Set(), Map(), ...